Dockerfile

```diff
@@ -7,6 +7,10 @@ USER root
 # dependencies relied upon to build native-extension gems etc
 RUN apk update
 RUN apk add libxml2-dev libxslt-dev build-base postgresql-dev tzdata
+# Fix incompatibility with slim tzdata from 2020b onwards
+RUN wget https://data.iana.org/time-zones/tzdb/tzdata.zi -O /usr/share/zoneinfo/tzdata.zi && \
```

Comment:
A few questions:

* Is there a link you can add which explains this in more detail, in case someone's wondering about it?
* Is this a temporary issue that will go away soon (i.e. that this step can be taken back out)
* how much does this slow the build by?

Reply:
1. Yep, the three links in the "This filters through to **Ruby** and **Rails** via the **tzinfo** gem" line of the description are links to explanations of details from the different perspectives. I could maybe be more explicit, true - I'll update the description with a long-form URL

2. Hopefully a future version of `tzdata` will fix it, at which point we can remove this line

3. A few seconds, not long